src/preload.ts

```diff
@@ -157,14 +165,15 @@ contextBridge.exposeInMainWorld('electron', {
     isWindows,
     isLinux,
   },
+  // Use this to access dynamic properties from the node side.
+  // INTENDED ONLY TO BE USED FOR TESTS.
```

Comment:
```suggestion
  /**
    * Use this to access dynamic properties from the node side.
    * {warning} INTENDED ONLY TO BE USED FOR TESTS.
  */
```

JSDoc comments so other devs can get this warning in their editors

Reply:
Nice one, I gotta keep that advice in mind